doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -96,28 +96,28 @@ After building is complete, proceed to trigger tests -
 
 Other frequently used/useful options are ``-d`` (or ``--distro``),
 ``--distroversion``, ``--filter-out``, ``--timeout``, ``flavor``, ``-rerun``,
-``-l`` (for limiting number of jobs) , ``-n`` (for how many times job would
-run). Run ``teuthology-suite --help`` to read description of these and every
-other options available.
+``-l`` (for limiting number of jobs) , ``-n`` (for how many times the job will
+run). Run ``teuthology-suite --help`` to read descriptions of these and other
+options.
 
 Testing QA changes (without re-building binaires)
 *************************************************
 
 While writing a PR you might need to test your PR repeatedly using teuthology.
-If you are making non-QA changes, you need to follow the standard process of
+If you are making non-QA changes, you must follow the standard process of
 triggering builds, waiting for it to finish and then triggering tests and
 wait for the result.
-But if changes you made are purely changes in qa/, you don't need rebuild the
+But if you made changes only in ``qa/``, you do not need to rebuild the
 binaries. Instead you can test binaries built for the ceph-ci branch and
-instruct ``teuthology-suite`` command to use a separate branch for running
+instruct the ``teuthology-suite`` command to use a separate branch for running
 tests.
 The separate branch can be passed to the command by using ``--suite-repo`` and
-``--suite-branch``. Pass the link to the GitHub fork where your PR branch exists
-to the first option and pass the PR branch name to the second option.
+``--suite-branch``. To the first option, pass the link to the GitHub fork where
+your PR branch exists.  To the second option, pass the PR branch name.
```

Comment:
I've added this sentence. Actually, I added a modified version of this sentence, with some numbered signposting so that the items in the two-item list in the second half of the sentence are clearly distinguished from one another.